lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -131,6 +132,16 @@ class Output
         }                                                                     \
     } while (false)
 
+#define ASSERT_ENUM(T, e)                                                     \
+    do                                                                        \
+    {                                                                         \
+        if ((int)(e) >= 0 && (e) < T::Max)                                    \
```

Comment:
This check will cause an assert if the enum IS in the correct range. You probably meant to check the opposite.

Reply:
This is a good case for T::Min on all the enum types.